src/main/resources/mixins.vanilla.chunkio.json

```diff
@@ -0,0 +1,17 @@
+{
+    "required": true,
+    "minVersion": "0.5.1",
+    "package": "org.spongepowered.server.mixin.chunkio",
+    "refmap": "mixins.vanilla.refmap.json",
+    "priority": 1112,
+    "server": [
+        "MixinAnvilChunkLoader_ChunkIO",
```

Comment:
@kashike yes, all custom toggle able mixins are suffixed in this fashion to avoid confusion between the potential several same targeted named mixins.